README.md

```diff
@@ -0,0 +1,74 @@
+| `mattermost_github_integration_secret`         | :heavy_multiplication_x: | `None`                               | Secret that is used to authenticate between github and your server. |
+| `mattermost_github_integration_server_hook`    | :heavy_multiplication_x: | `/`                                  | The relativ url where the server is listing.                        |
+| `mattermost_github_integration_server_address` | :heavy_multiplication_x: | `0.0.0.0`                            | Adress of the server                                                |
+| `mattermost_github_integration_server_port`    | :heavy_multiplication_x: | `5000`                               | Port under which the server listens to webooks from github          |
+| `mattermost_github_integration_username`       | :heavy_multiplication_x: | `Github`                             | Name under which username the post should show up in mattermost     |
+| `mattermost_github_integration_icon_url`       | :heavy_multiplication_x: | ` `                                  | Url to icon file which should show up in mattermost                 |
+| `mattermost_github_integration_webhook_urls`   | :heavy_multiplication_x: | `[]`                                 | List of webhooks to post to. See below for more information         |
+| `mattermost_github_integration_show_avatars`   | :heavy_multiplication_x: | `true`                               | show github avatars in the message that is posted                   |
+| `mattermost_github_integration_ignore_actions` | :heavy_multiplication_x: | `[]`                                 | List of actions to ignore- See below for more information           |
+
+### Webhooks
+
+| Name      |      Required      | Default | Description                   |
+|:----------|:------------------:|:--------|:------------------------------|
+| `name`    | :heavy_check_mark: |         | Name of the repository/team where this information should go to |
+| `url`     | :heavy_check_mark: |         | Url of the mattermost webhook |
+| `channel` | :heavy_check_mark: |         | Channel id like town-square from mattermost. This is where the information gets posted to. |
+
+When using `name` you can add `default` as a variable where everything is send to if it finds no other match. Also you can use `teamname` or `teamname/repositoryname` to have a better controll over the information like `stuvusIT` or `stuvusIT/mattermost-github-integration`
+
+### Ignore actions
+| Name       |      Required      | Default | Description                                             |
+|:-----------|:------------------:|:--------|:--------------------------------------------------------|
+| `category` | :heavy_check_mark: |         | Path to install the server on                           |
+| `actions`  | :heavy_check_mark: |         | List of actions to be ignored e.g `labeled`, `assigned` |
```

Comment:
Is there a complete list somewhere? Also, what kind of description is this for a category?

Reply:
fixed